minimum.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+// find mix or max number between two defined numbers
+
+func main() {
+	a := math.Min(100, 20)
+	fmt.Println(a)
+
+	b := math.Max(100, 20)
+	fmt.Println(b)
+}
```

Comment:
I feel this file should be renamed to `math.go` and should contain frequently used math methods in go

Reply:
@ozombo

Reply:
I'd update this.

Reply:
Take a look @ichtrojan